### DISCUSSION TITLE
Need To Keep Paint, GameBar and Xbox Apps.

### DISCUSSION THREAD

**Discussion by KedarWolf** [memstechtips/UnattendedWinstall#98] [Q&A] [answered]
I want to use the unattend file, but need to keep Paint, GameBar and the Xbox apps like:

Microsoft.XboxApp | Xbox
Microsoft.XboxGameOverlay | Xbox Game Bar
Microsoft.XboxGamingOverlay | Xbox Game Bar
Microsoft.XboxIdentityProvider | Xbox Identity Provider
Microsoft.XboxSpeechToTextOverlay

How can I do this by editing the unattend file?

**Comment by KedarWolf** [comment, marked as answer]
I'm thinking I can just use the answer file as is, and install GameBar and Paint from the Microsoft store. Just make sure the related services are enabled.

**Reply by memstechtips** [reply]
Yes, that will work.

Alternatively, use a code editor and do a search for the related things you want to keep. Remove those entries from the file and save it again.

**Comment by memstechtips** [comment]
Hey there and sorry for the late reply,

Since the creation of this issue I have released a new version of the XML file and a major update to [Winhance](https://winhance.net/), I recommend that you watch [this video](https://youtu.be/lrq3ph3xi50) which provides a full demo of the current version along with how to use the new version of WIMUtil and how to install the drivers etc. You now also have the ability to create an autounattend.xml file based on your selections in Winhance, so you have more control over which apps are removed and which settings are applied.

I have also created [this video](https://youtu.be/sCd1TI6AbOI) about v3.0.0 of UnattendedWinstall, I suggest that you watch it too if you're interested.

I'm closing this issue now, but if you need more help with the same issue, feel free to reopen it.****
